### PR TITLE
spdx-utils: Miscellaneous fixes

### PR DESCRIPTION
**Pull request by fviernau**
See the individual commits.

### CODE REVIEW

File: spdx-utils/src/main/kotlin/SpdxUtils.kt [hunk 1]
```diff
@@ -176,7 +176,7 @@ internal fun String.isSpdxExpression(): Boolean =
 
 /**
  * Return true if and only if this String can be successfully parsed to an [SpdxExpression] or if it equals
- * [org.ossreviewtoolkit.spdx.model.SpdxConstants.NONE] or [org.ossreviewtoolkit.spdx.model.SpdxConstants.NOASSERTION].
+ * [SpdxConstants.NONE] or [SpdxConstants.NOASSERTION].
```
Comment (sschuberth) [inline spdx-utils/src/main/kotlin/SpdxUtils.kt:179]: Commit message: s/kDoci/KDoc/

File: spdx-utils/src/main/kotlin/model/SpdxExternalReference.kt [hunk 2]
```diff
@@ -26,7 +26,7 @@ import com.fasterxml.jackson.annotation.JsonValue
  *  References an external source of additional information, metadata, enumerations, asset identifiers, or downloadable
  *  content believed to be relevant to the Package.
  */
-class SpdxExternalReference(
+data class SpdxExternalReference(
```
Comment (sschuberth) [inline spdx-utils/src/main/kotlin/model/SpdxExternalReference.kt:29]: Commit message: For consistency with what? I assume you mean with the other SPDX classes, so please say "For consistency with the other SPDX classes in the model package."

File: spdx-utils/src/main/kotlin/SpdxUtils.kt (hunk 1 above)
Comment (fviernau) [reply spdx-utils/src/main/kotlin/SpdxUtils.kt:179]: Done.

File: spdx-utils/src/main/kotlin/model/SpdxExternalReference.kt (hunk 2 above)
Comment (fviernau) [reply spdx-utils/src/main/kotlin/model/SpdxExternalReference.kt:29]: Done.

**Pull request merged**